.github/workflows/build.yml

```
@@ -95,5 +95,5 @@ jobs:
       - name: Update Homebrew/AUR
```

```suggestion
      - name: Update AUR package
```
Since we're going to do the homebrew one with a separate action.